user-guide/Getting_started/Installing_a_DMA/Using_a_pre_installed_DataMiner_Virtual_Hard_Disk.md

```diff
@@ -127,6 +127,39 @@ Follow the below steps to configure your DataMiner Agent:
 > [!IMPORTANT]
 > For security reasons, we strongly recommend creating a second user and disabling the built-in administrator account once the setup is complete. See [Managing users](xref:Managing_users).
 
+### WSL management
+
+If you chose for the Self-hosted - Local storage you are running a WSL container for your storage. This can be managed if you like.
+To connect to the WSL you will need to open a command prompt with the Administrator user (or the user that is going through the wizard) and run the `wsl` command.
+It should log in automatically and connect to the WSL container.
+
+   > [!NOTE]
+   > WSL always runs under a user, it is not possible to reach a WSL container from another user account.
+
+#### Cassandra
+
+Cassandra is installed with the following user:
+* cassandra
+* DataMiner123!
+
+When Cassandra is running you can change the password with `cqlsh -u cassandra -p DataMiner123!` and enter the following query: `alter role cassandra with password = 'MyNewPassword123!';`
+Exit *cqlsh* with `exit`.
+
+#### Opensearch
+
+Opensearch admin user:
+* admin
+* DataMiner123!
+
+You can change the admin password by following these steps:
```

Comment:
Could you adjust the procedure below this so it's more clear to the user what they should do? This means changing it to a numbered list instead of a bulleted list, and more importantly, not just mentioning commands, but actually giving instructions that mention what the user should do and how.